ray-operator/controllers/ray/common/pod.go

```diff
@@ -646,13 +646,20 @@ func setContainerEnvVars(pod *v1.Pod, rayContainerIndex int, rayNodeType rayiov1
 			}
 		}
 	}
+	if !envVarExists(RAY_GCS_RPC_SERVER_RECONNECT_TIMEOUT_S, container.Env) && rayNodeType == rayiov1alpha1.WorkerNode {
+		// If GCS FT is enabled and RAY_GCS_RPC_SERVER_RECONNECT_TIMEOUT_S is not set, set the worker's
+		// RAY_GCS_RPC_SERVER_RECONNECT_TIMEOUT_S to 600s. If the worker cannot reconnect to GCS within
+		// 600s, the Raylet will exit the process. By default, the value is 60s, so the head node will
+		// crash if the GCS server is down for more than 60s. Typically, the new GCS server will be available
+		// in 120 seconds, so we set the timeout to 600s to avoid the worker nodes crashing.
```

Comment:
120 seconds works in my local Kind cluster, but it may not work in a production Kubernetes cluster. Users can easily change the value by setting an environment variable. In addition, it is possible for some Pods on public cloud to experience network disconnections for several minutes. That's why I decided to select a higher timeout.

Reply:
Gentle ping @wilsonwang371 

Does this make sense to you?